### PR TITLE
IOS build settings

### PR DESCRIPTION
**Pull request by Jaqueeee**

### CODE REVIEW

File: CMakeLists_IOS.txt [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# This file is based off of the https://code.google.com/archive/p/ios-cmake/
```
Comment (Jaqueeee) [inline CMakeLists_IOS.txt]: @fluffypony 
Not sure of best practice here. 
https://code.google.com/archive/p/ios-cmake/ is licensed under "New BSD License"
I've altered the file a bit to make it work with monero. Should i add Monero copyright to header?

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -96,23 +96,33 @@ function (monero_add_library name)
...
+else()
+    add_subdirectory(common)
+    add_subdirectory(crypto)
+    add_subdirectory(ringct)
+    add_subdirectory(cryptonote_basic)
+    add_subdirectory(cryptonote_core)
+    add_subdirectory(mnemonics)
+    add_subdirectory(wallet)
+    add_subdirectory(cryptonote_protocol)
+endif()
```
Comment (moneromooo-monero) [inline src/CMakeLists.txt]: I'd keep the common parts out, it simplifies future changes.

File: CMakeLists_IOS.txt (hunk 1 above)
Comment (moneromooo-monero) [inline CMakeLists_IOS.txt]: If the licenses are compatible, and you changed the code, you should add, eg: Copyright 2017 the Monero project. Or maybe Portions copyright 2017 the Monero project. The main copyright notice stays, you just append. It is much simpler to licence the modifications under the same licence as the original.

Comment (Jaqueeee) [inline CMakeLists_IOS.txt]: Added copyright notice

File: src/CMakeLists.txt (hunk 2 above)
Comment (Jaqueeee) [inline src/CMakeLists.txt]: Done

**Pull request merged**